### PR TITLE
slight modernization of dapp.tools

### PR DESCRIPTION
**Pull request by MrChico**
Things are generally horrendously out of date. This is just a quick pass, focusing on not downplaying hevm as much

### CODE REVIEW

File: seth/index.html [hunk 1]
```diff
@@ -29,6 +23,7 @@ <h1>
 <strong>call</strong>       call a contract without updating the blockchain
 <strong>calldata</strong>   pack a signature and a list of arguments into calldata
 <strong>code</strong>       print the bytecode of an object on the blockchain
+<strong>debug</strong>      step through transaction in hevms interactive debugger
```
Comment (asymmetric) [inline seth/index.html]:
```suggestion
<strong>debug</strong>      step through a transaction in hevm's interactive debugger
```

File: hevm/index.html [hunk 2]
```diff
@@ -26,34 +21,61 @@ <h1>
...
-<strong>$ hvem dapp-test</strong>            run dappsys unit tests
-<strong>$ hvem interactive</strong>          enter the interactive debugger
-
-OPTIONS
-
---json-file      run against a specific output file
---rpc            fetch external contracts via rpc from the latest block
-</pre>
+<code>
+hevm -- Ethereum evaluator
```
Comment (asymmetric) [inline hevm/index.html]: Should we mention Haskell, to explain the "h"?

File: hevm/index.html [hunk 3]
```diff
@@ -26,34 +21,61 @@ <h1>
...
+  interactive
+  bc-test
+  compliance
+  flatten
+  emacs
+  version
+  rlp
+  abiencode
+  merkle-test
+  strip-metadata
```
Comment (asymmetric) [inline hevm/index.html:45]: I don't know if this is useful to have on the website, but I'm fine if you want to add it.

File: hevm/index.html (hunk 2 above)
Comment (MrChico) [inline hevm/index.html]:
```suggestion
hevm -- Ethereum evaluator written in Haskell
```

**Review by asymmetric — APPROVED** [review]

**Pull request merged**